about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="assets/stylesheets/about.css">
+  <link rel="stylesheet" href="assets/stylesheets/popup.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <div class="mobile-popup-menu">
+    <div id="menu-mobile">
+      <img id="close-button" src="./assets/images/close-icon.svg" alt="close button">
+      <ul class="contain-menu-link">
+        <li><a class="menu-link" href="./index.html">Home</a></li>
+        <li><a class="menu-link" href="./about.html">About</a></li>
+        <li><a class="menu-link" href="#">Program</a></li>
+        <li><a class="menu-link" href="#">Join</a></li>
+        <li><a class="menu-link" href="#">Sponsor</a></li>
+        <li><a class="menu-link" href="#">News</a></li>
+      </ul>
+      <hr class="menu-red-line">
+    </div>
+  </div>
```

Comment:
- [ ] It would be a good thing to have all your navigation bar links and texts inside a header tag. This ensures that your project code base is semantically correct and free from errors.